code/modules/station_goals/bluespace_tap.dm

```diff
@@ -0,0 +1,359 @@
+//Station goal stuff goes here
+/datum/station_goal/bluespace_tap
+	name = "Bluespace Tap"
+	var/goal = 500000
+
+/datum/station_goal/bluespace_tap/get_report()
+	return {"<b>Bluespace Mining Tap Experiment</b><br>
+	Our search for new resources to exploit is nearing a tremendous break-through. One of our research stations in a nearby sector has recently finished a prototype for a device called a Bluespace Tap.
```

Comment:
might be a bit wordy, but I wanted people to know what it is and how it works. Might instead include a 'manual' with the crate? Thoughts?